build.js

```diff
@@ -1,55 +1,75 @@
 const StyleDictionary = require("style-dictionary").extend("config.json");
-const ChangeCase = require('change-case')
+const ChangeCase = require("change-case");
```

Comment:
Prettier formatting here updating the quotes style

Reply:
I ran into the same thing when I had my MR yesterday 🤪